docs/ReleaseNotes.md

```diff
@@ -54,7 +54,7 @@ The `FDBDatabase::getReadVersion()` method has been replaced with the `FDBRecord
 * **Performance** Improvement 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Feature Add assertions into record layer [(Issue #925)](https://github.com/FoundationDB/fdb-record-layer/issues/925)
```

Comment:
```suggestion
* **Feature** Add assertions into record layer [(Issue #925)](https://github.com/FoundationDB/fdb-record-layer/issues/925)
```

Hmm, this also feels like it should be more descriptive. Like, "added assertions about key and value size" or something. I think keeping it more general ("assertions") is probably fine in the code, though, to future proof it for more kinds of assertions.

Reply:
Fixed.